app/Exceptions/Handler.php

```diff
@@ -215,7 +215,7 @@ protected function convertExceptionToArray(\Throwable $e, array $override = []):
                         ->map(fn ($trace) => Arr::except($trace, ['args']))
                         ->all(),
                     'previous' => Collection::make($this->extractPrevious($e))
-                        ->map(fn ($exception) => $e->getTrace())
+                        ->map(fn () => $e->getTrace())
```

Comment:
```suggestion
                        ->map(fn ($exception) => $exception->getTrace())
```

Reply:
yeah that's what i thought but i wasn't sure